lib/controller/python/controller/brake.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 1996-2022 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ctypes
+from controller.constants import constant
+from controller.device import Device
+from controller.wb import wb
+from typing import Union
+
+
+class Brake(Device):
+    ROTATIONAL = constant('ROTATIONAL')
+    LINEAR = constant('LINEAR')
+
+    def __init__(self, name: Union[str, int]):
+        super().__init__(name)
+
+    def setDampingConstant(self, c: float):
+        wb.wb_brake_set_damping_constant(self._tag, ctypes.c_double(c))
+
+    dampingConstant = property(fset=setDampingConstant)
+
+    def getType(self) -> int:
+        return self.type
+
+    def getMotor(self):
+        return self.motor
+
+    def getPositionSensor(self):
+        return self.position_sensor
+
+    @property
+    def motor(self):
+        from controller.motor import Motor
+        tag = wb.wb_brake_get_motor(self._tag)
+        return None if tag == 0 else Motor(tag)
+
+    @property
+    def position_sensor(self):
+        from controller.position_sensor import PositionSensor
+        tag = wb.wb_brake_get_position_sensor(self._tag)
+        return None if tag == 0 else PositionSensor(tag)
```

Comment:
I did not added them here because it would cause a circular dependency problem which I was unable to solve.
However, these type annotations are not mandatory, they are just a help for the user to know which data type is returned.